Goal: Cancel login timeout after authentication so that patient people
      timing out on network directory services can log in with local
      accounts.
Fixes: #107148

Status wrt upstream: It should be forwarded to upstream.

Index: shadow-4.0.12/src/login.c
===================================================================
--- shadow-4.0.12.orig/src/login.c	2005-08-17 11:50:55.000000000 +0200
+++ shadow-4.0.12/src/login.c	2005-08-17 11:50:56.000000000 +0200
@@ -736,6 +736,7 @@
 			}
 
 			/* We don't get here unless they were authenticated above */
+			alarm (0);
 			retcode = pam_acct_mgmt (pamh, 0);
 
 			if (retcode == PAM_NEW_AUTHTOK_REQD) {
